scripts/check-commit-messages.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+for commit in $(git rev-list origin/HEAD..HEAD); do
+
+    commit_msg=$(git log --format=%B -n 1 "$commit")
+
+    # Skip validation in case of fixup and revert commits
+    if echo "$commit_msg" | grep -qE "^(Revert|fixup! )"; then
+    continue
+    fi
+
+    if ! echo "$commit_msg" | grep -qE "^(build|ci|docs|feat|fix|perf|refactor|style|test|chore|revert)(\([a-z\-]+\))?: "; then
```

Comment:
please allow also multiple scopes like `chore(suite,e2e,foo): blabla`
```suggestion
    if ! echo "$commit_msg" | grep -qE "^(build|ci|docs|feat|fix|perf|refactor|style|test|chore|revert)(\([a-z,\-]+\))?: "; then
```

Reply:
Done in 6c4ca24c3535c8a0b7eccaabfd22c1993418e9a1. I also allowed for spaces. The [Conventional commits specification](https://www.conventionalcommits.org/en/v1.0.0/#specification) does not cover this.

Readmes also updated.